### ISSUE TITLE
Split carets

### ISSUE THREAD

**Issue by MetRonnie** [PanAeon/vscode-tmgrammar-test#7]
Possible feature request: allowing split `^`s, e.g.
```javascript
let foo, bar;
//  ^^^  ^^^ variable.other.readwrite.js
```
**Note:** Currently, the example above, or even something like
```javascript
const foo = 2;
//   ^^      ^ non.existant.scope
```
doesn't raise an error or anything,

**Comment by PanAeon** [comment]
I'll take a look tomorrow for this.

**Comment by PanAeon** [comment]
@MetRonnie yes, you are right.  I thought that it would be OK to ignore invalid comments so I put a strict regex that silently filters invalid assertions, including split `^`. 
LOL, I even wrote a test 'should not parse multiple accent groups'.
Just released `0.0.9` where I've added support for split carets, but also show an error message when a line starts with `<comment> ^^^` but is unparsable.
